binding/react-native/android/src/main/java/ai/picovoice/reactnative/porcupine/PorcupineModule.java

```diff
@@ -132,7 +140,7 @@ public void process(String handle, ReadableArray pcmArray, Promise promise) {
       int result = porcupine.process(buffer);
       promise.resolve(result);
     } catch (PorcupineException e) {
-      promise.reject(e.toString());
+      promise.reject(e);
```

Comment:
again, there must be a conversion we can do to bring the right exception to the RN layer